### DISCUSSION TITLE
How to Create a Multi-page Website

### DISCUSSION THREAD

**Discussion by ellefon** [community/community#35718] [New to GitHub] [answered]
Hello, this is my first week learning HTML and Github. I've been trying to create a two-page website by including two HTML files in my repo; my index file and a file labeled "second." I've learned that you can link two pages and create a hyperlink by using: < a href="html/second.html">Link to Second Page < /a > (added spaces for clarity). However, when I visit my Github site, clicking the hyperlink results in a 404. How do solve this? Thanks.

**Comment by airtower-luna** [comment, marked as answer]
`<a href="html/second.html">` creates a relative link to `second.html` within a directory called `html`. If you have the files right next to each other there won't be such a directory, and you don't need it.

A relative link (one not starting with a URL scheme like `https://`, or a slash `/`) means always to start looking in the same directory as the file in which the link is. So if the files `index.html` and `second.html` are in the same directory, this will work:

```html
<a href="second.html">link text</a>
```

See also: [What is a URL?](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL), in particular the [Absolute URLs vs relative URLs](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL#absolute_urls_vs_relative_urls) section.

**Reply by ellefon** [reply]
Thank you so much! That solved it, I appreciate the help.

**Reply by zanaart** [reply]
<a href="second.html">link text</a>

**Comment by MitkoVtori** [comment]
It's even easier by doing this:
`<a href="./name">link text</a>`
or for main page
`a href="./">link text</a>`
Those will remove the .html from your link, for example:
your link **probably** is name/somethinglalalal.html and your second page is name/somethingalalalal/second%20page.html
This will remove your .html and the link will look cooler.

**Reply by ellefon** [reply]
Good to know! Appreciate the help.

**Comment by queenofcorgis** [comment]
Thanks @airtower-luna and @MitkoVtori for chiming in - love that there are multiple perspectives!
